src/vs/workbench/services/untitled/common/untitledEditorService.ts

```diff
@@ -130,7 +154,7 @@ export class UntitledEditorService implements IUntitledEditorService {
 			.map((i) => i.getResource());
 	}
 
-	public createOrGet(resource?: URI, modeId?: string): UntitledEditorInput {
+	public createOrGet(resource?: URI, modeId?: string, restoreResource?: URI): UntitledEditorInput {
```

Comment:
@Tyriar can you explain again why we cannot do something similar as with files (`this.backupFileService.doesTextFileHaveBackup`)